core/src/main/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolver.java

```diff
@@ -1026,6 +1031,17 @@ private Object convert(@Nullable String value) {
         if (value == null) {
             return defaultOrException();
         }
+        if (value.isEmpty()) {
+            logger.warn("Parameter is present but the value is missing: " + httpElementName);
```

Comment:
As it is an expected behavior, we can remove the warning.

Reply:
I added this warning because @ikhoon commented in #5515 :
>In addition to the bugfix, we also need to improve the error message when an incorrect request is made so that users can easily identify the problem.

How about warning only if the type is not `String`?

```suggestion
            if (String.class.isAssignableFrom(elementType)) {
                return value;
            }
            if (elementType.isPrimitive()) {
                throw new IllegalArgumentException("Cannot set null to primitive type parameter: " +
                                                   httpElementName);
            }
            logger.warn("Parameter is present but the value is missing: " + httpElementName);
            return null;
```